README.md

```diff
@@ -0,0 +1,87 @@
+# vo-models
+
+`vo-models` an open-source project to provide Python models and OpenAPI specifications for [IVOA](https://www.ivoa.net/) service protocols.
+
+The project is designed to be used by IVOA members, service implementors, and developers to help facilitate the development of IVOA-compliant services and clients.
+
+## Features
+
+- **Pydantic-xml Models:** The project includes Python models for IVOA protocols, using [pydantic-xml](https://github.com/dapper91/pydantic-xml). Based on [Pydantic](https://docs.pydantic.dev/latest/), these models describe transactions for an IVOA protocol, such as UWS, and feature automatic validation, parsing and serialization of XML data for use with Python clients and web frameworks.
+
+- **OpenAPI Specifications:** The project includes OpenAPI definitions for IVOA protocols. The use of OpenAPI provides a standardized and machine-readable way to describe the IVOA protocols. OpenAPI specifications offer benefits such as automatic documentation generation, and automatic client and server code generation.
+
+- **Expandability:** The project is designed with future expansion in mind. Plans include extending the schema and models to cover other IVOA standards.
+
+## Protocols
+
+The following IVOA protocols are currently supported / under development:
+
+- **UWS (Universal Worker Service) version 1.1:**
+  - Active development:
+    - OpenAPI Models
+    - Pydantic-XML Models
+  - Planned:
+    - OpenAPI Service Definition
+
+## Installation
+
+The latest version of the project can be installed from PyPI:
+
+```bash
+pip install vo-models
+```
+
+### Conda
+
+To install the project using Conda, you can use the provided environment file:
+
+```bash
+git clone https://github.com/spacetelescope/vo-models.git
+cd vo-models
+conda env create -f environment.yml
+conda activate vo-models
+pip install -r requirements.txt
+pip install .
+```
+
+For active development, install the project in development mode:
+
+```bash
+pip install -e .
+```
+
+### Pip
+If you prefer to use pip, you can install the project directly from the repository:
+
+```bash
+git clone https://github.com/spacetelescope/vo-models.git
+cd vo-models
+pip install -r requirements.txt
+pip install .
+```
```

Comment:
I think you can probably just remove this section now?

it's also confusing since you mention installing with pip already before you get to this section